photosynthesis_metrics/__init__.py

```diff
@@ -8,3 +8,6 @@
 from .isc import IS, inception_score
 from .vif import VIFLoss, vif_p
 from .brisque import BRISQUELoss, brisque
+
+
+__version__ = "0.4.0"
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names) the `__version__` should be defined before import statements.

Reply:
Hmm it means that [even pip does it the wrong way](https://github.com/pypa/pip/blob/master/src/pip/__init__.py) 😄
Ok, I will change that